Integrations/FireEyeHelix/FireEyeHelix.py

```diff
@@ -0,0 +1,1643 @@
+    list_id = args.get('list_id')
+    raw_response = client.update_list_item(**args)
+    if raw_response:
+        item_id = raw_response.get('id')
+        title = f'{INTEGRATION_NAME} - List item {item_id} from list {list_id} was updated successfully'
+        context_entry = build_transformed_dict(raw_response, LIST_ITEM_TRANS)
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}List(val.ID && val.ID === {list_id}).Item(val.ID === obj.ID)': context_entry
+        }
+        human_readable = tableToMarkdown(title, context_entry)
+        # Return data to Demisto
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - Could not update list item.', {}, raw_response
+
+
+def remove_list_item_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Updates a list item. return outputs in Demisto's format
+
+    Args:
+        client: Client object with request
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs
+    """
+    list_id = args.get('list_id')
+    item_id = args.get('item_id')
+    raw_response = client.remove_list_item(list_id=list_id, item_id=item_id)
+    return f'{INTEGRATION_NAME} - Removed item {item_id} from list {list_id} successfully', {}, raw_response
+
+
+def get_list_items_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Fetches list items
+
+    Args:
+        client: Client object with request
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs
+    """
+    list_id = args.get('list_id')
+    raw_response = client.get_list_items(list_id, args.get('offset'))
+    results = raw_response.get('results')
+    if results:
+        title = f'{INTEGRATION_NAME} - List items for list {list_id}'
+        context_entry = build_transformed_dict(results, LIST_ITEM_TRANS)
+        count = demisto.get(raw_response, 'meta.count')
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}List(val.ID && val.ID === {list_id}).Item(val.ID === obj.ID)': context_entry,
+            f'{INTEGRATION_CONTEXT_NAME}List(val.ID && val.ID === {list_id}).Count(val.Count)': count
+        }
+        human_readable = tableToMarkdown(title, context_entry)
+        # Return data to Demisto
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - No items were found for list {list_id}.', {}, raw_response
+
+
+def list_sensors_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Lists all sensors and return outputs in Demisto's format
+
+    Args:
+        client: Client object with request
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs
+    """
+    raw_response = client.list_sensors(**args)
+    sensors = raw_response.get('results')
+    if sensors:
+        title = f'{INTEGRATION_NAME} - List sensors:'
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}.Sensor(val.id && val.ID === obj.id)': sensors  # TODO: Edit this
```

Comment:
TODO?

Reply:
This is to be decided by me and @Arsenikr in today's meeting, as we don't have access to data about sensors.